### ISSUE TITLE
Need a way to handle ambiguous mapping between media and talks

### ISSUE THREAD

**Issue by srogers** [srogers/conferences#137]
Need a way to manage the case where a publication exists, but we don't know the exact source, and there are multiple talks. It would be nice to have media entered only once, so the counts of publications would be meaningful. 

Maybe a presentation_publications with has_many => through and a way to mark a relationship as being the known true recording. 

For example:
https://objectivistconferences.info/presentations?search_term=contextual
We don't know which conference the media belongs to. Entering it twice inflates the counts of talks. Not entering it makes it seem like no media exists if you're looking at the 1995 talk.

**Comment by srogers** [comment]
The down-side - this would require significant changes to data entry. "Manage Publications" could pre-find publications with names exactly the same as the presentation (or very similar) and allow them to be associated with one click. 

But the editor would also need a way to conveniently add  a publication and tie it in. 

Orphan publications is the way to go for tapes of unknown origin, instead of unknown presentations - require a conference on presentations.

**Comment by srogers** [comment]
Deployed
